koku/koku/middleware.py

```diff
@@ -105,35 +101,6 @@ def __init__(self, dikt):
         super().__init__(data)
 
 
-class KokuTenantSchemaExistsMiddleware(MiddlewareMixin):
-    """A middleware to check if schema exists for Tenant."""
-
-    def process_exception(self, request, exception):
-        if isinstance(exception, (Tenant.DoesNotExist, ProgrammingError)):
```

Comment:
we may need this? and remove just the inner if block?

Reply:
Yes, thank you @esebesto

I got a little too heavy handed with the removal of the middleware. 🙇